.github/workflows/test_e2e_web_flow.yaml

```
@@ -9,8 +9,17 @@ jobs:
   changes:
     runs-on: ubuntu-latest
     outputs:
+      permissions: ${{ steps.check-permissions.outputs.permissions }}
```

actionlint (1.7.7)

property "check-permissions" is not defined in object type {filter: {conclusion: string; outcome: string; outputs: object}}

(expression)